bundles/admin/admin-layereditor/view/AdminLayerForm/AdditionalTabPane/SelectedTime.jsx

```diff
@@ -7,7 +7,7 @@ import { StyledFormField } from '../styled';
 export const SelectedTime = ({ layer, controller }) => {
     const value = layer.params ? layer.params.selectedTime : '';
     const { capabilities } = layer;
-    if (!Array.isArray(capabilities.times)) {
+    if (!capabilities.times) {
```

Comment:
This has nothing to do with announcements and the array check is needed to prevent js error. Please rollback this change.